### ISSUE TITLE
MSVC 2022 cannot build this project

### ISSUE THREAD

**Issue by Pomax** [github/cmark#359]
Using MSVC 2022 Community Edition:

```
PS  D:\temp\cmark-gfm> nmake

Microsoft (R) Program Maintenance Utility Version 14.38.33134.0
Copyright (C) Microsoft Corporation.  All rights reserved.

makefile(4) : fatal error U1001: syntax error : illegal character '?' in macro
Stop.
```

**Comment by mklabs** [comment]
Hi @Pomax I got the exact same error when running the VS command line using powershell.

If you did the same, try using cmd.exe for the shell, I was able to build via cmd:

![image](https://github.com/user-attachments/assets/51b56378-25b4-4bef-a6c8-0dfe5c869bc9)

choosing the command prompt for Visual Studio instead of powershell.

![image](https://github.com/user-attachments/assets/2e7c1288-ba50-4361-ae38-b12766649e4e)

**Comment by Pomax** [comment]
Ah! Yes, of course, powershell doesn't support normal shell things like `&&`.

Hopefully the readme can be edited to point out that Windows users need to use `cmd`

**Comment by mklabs** [comment]
I ended up using the `x64 Native Tools Command Prompt for VS 2022` from the Windows Start Menu, instead of the one from Visual Studio Tools menu. This should come with a visual studio install.

![image](https://github.com/user-attachments/assets/ea72a54a-fffa-4d9e-ba94-12484e30cf2c)

I needed to compile the static lib for x64 architecture and couldn't get nmake to do so with the developer command prompt out of the tools menu in VS.

---

In summary:

1. From windows start menu, search for and open `x64 Native Tools Command Prompt for VS 2022`. This should start a windows terminal with cmd shell and environment set to x64.
2. cd to the root of cmark-gfm repo
3. Run `nmake`
4. The binaries and static lib can be found in build/src and build/extensions

Hope that helps :)

**Comment by Pomax** [comment]
awesome sleuthing, thanks! Hopefully others will find this information too.
